initrd-builder/initrd_builder.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 HyperHQ Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+script_name="${0##*/}"
+script_dir="$(dirname $(readlink -f $0))"
+
+if [ -n "$DEBUG" ] ; then
+	set -x
+fi
+
+SCRIPT_NAME="${0##*/}"
+INITRD_IMAGE="${INITRD_IMAGE:-kata-initrd.img}"
+
+die()
+{
+	local msg="$*"
+	echo "ERROR: ${msg}" >&2
+	exit 1
+}
+
+OK()
+{
+	local msg="$*"
+	echo "[OK] ${msg}" >&2
+}
+
+info()
+{
+	local msg="$*"
+	echo "INFO: ${msg}"
+}
+
+usage()
+{
+	error="${1:-0}"
+	cat <<EOT
+Usage: ${SCRIPT_NAME} [options] <rootfs-dir>
+	This script will create a Kata Containers initrd image file based on the
+	<rootfs-dir> directory.
+
+Options:
+	-h Show this help
+	-o path to generate image file ENV: INITRD_IMAGE
```

Comment:
Nit: You could also include the default value of `$INITRD_IMAGE` here.

Reply:
Missing verb, what does the -o option do? Does it set the output path, change it, or store it? I assume it sets the path. Consider changing to:
-o Set the path where the generated image file is stored. Default: the path stored in the environment variable INITRD_IMAGE